packages/ui/src/utils/UiHelperUtils.ts

```diff
@@ -72,5 +72,19 @@ export const UiHelperUtil = {
     }
 
     return regex.number.test(character)
+  },
+
+  getColorTheme(theme: ThemeType | undefined): ThemeType {
```

Comment:
```ts
getColorTheme(theme?: ThemeType)
```

Also, we don't do manual return types i.e. no `: ThemeType` as everything should be inherited by ts and fail down the line. If return type here is resolving to `string`, you can use `as const`